resources/views/components/molecules/_m-listing----article.blade.php

```diff
@@ -3,24 +3,30 @@
         @if (!isset($hideImage) || (isset($hideImage) && !($hideImage)))
             <span class="m-listing__img{{ (isset($imgVariation)) ? ' '.$imgVariation : '' }}{{ ($item->videoFront) ? ' m-listing__img--video' : '' }}"{{ (isset($variation) and strrpos($variation, "--hero") > -1 and !$item->videoFront) ? ' data-blur-img' : '' }}>
                 @if (isset($image) || $item->imageFront('hero'))
-                    @component('components.atoms._img')
-                        @slot('image', $image ?? $item->imageFront('hero'))
-                        @slot('settings', $imageSettings ?? '')
-                        @slot('class', 'img-hero-desktop')
-                    @endcomponent
-                    @component('components.atoms._img')
-                        @slot('image', $imageMobile ?? $item->imageFront('mobile_hero') ?? $item->imageFront('hero'))
-                        @slot('settings', $imageSettings ?? '')
-                        @slot('class', 'img-hero-mobile')
-                    @endcomponent
+                    @if (isset($isHero) && $isHero)
```

Comment:
It doesn't make much difference, but you could use the null coalesce operator here, as well:
```
@if ($isHero ?? false)
```

Reply:
@zachgarwood yeye this is a good change thank youuu